bp_top/test/rom/src/env/p/riscv_test.h

```diff
@@ -182,34 +182,11 @@ reset_vector:                                                           \
 //-----------------------------------------------------------------------
 // Pass/Fail Macro
 //-----------------------------------------------------------------------
-
-#define RVTEST_PASS                                                     \
-        fence;                                                          \
-        li t0, 0xc00dead0;                                              \
-        mv a0, TESTNUM;                                                 \
-        li a1, 0x0000ffff;                                              \
-        and a0, a0, a1;                                                 \
-        sw a0, 0(t0);                                                   \
-                                                                        \
-        fence;                                                          \
-        li TESTNUM, 1;                                                  \
-        ecall
-
 #define TESTNUM x28
-#define RVTEST_FAIL                                                     \
-        fence;                                                          \
-        li t1, 0xc00dead0;                                              \
-        mv a0, TESTNUM;                                                 \
-        li a1, 0xffff0000;                                              \
-        or a0, a0, a1;                                                  \
-        sw a0, 0(t1);                                                   \
-                                                                        \
-        fence;                                                          \
-1:      beqz TESTNUM, 1b;                                               \
-        sll TESTNUM, TESTNUM, 1;                                        \
-        or TESTNUM, TESTNUM, 1;                                         \
-        ecall
 
+#define RVTEST_PASS li t0, 0x0000beef; ecall
```

Comment:
Hmm, we don't have the infrastructure to support this ecall yet.  I would keep the old macros for now

Reply:
Yeah that call doesn't do anything, the tracegen stops the program on the register write